spdx/parsers/rdfbuilders.py

```diff
@@ -184,19 +188,26 @@ class PackageBuilder(tagvaluebuilders.PackageBuilder):
     def __init__(self):
         super(PackageBuilder, self).__init__()
 
-    def set_pkg_chk_sum(self, doc, chk_sum):
+    def set_pkg_checksum(self, doc, checksum: Checksum | Dict):
```

Comment:
This type hint only works for 3.7+, which we cannot assume yet

Reply:
fixed